src/main/webapp/app/localci/build-queue/build-queue.component.html

```diff
@@ -879,3 +879,46 @@ <h5 class="my-0">
         </button>
     </div>
 </ng-template>
+
+<ng-template #buildLogsModal let-modal>
+    <div class="modal-header">
+        <h5 class="modal-title">
+            <span jhiTranslate="artemisApp.buildQueue.logs.title"></span>
+            <span> {{ this.displayedBuildJobId }}</span>
+        </h5>
+        <button type="button" class="btn-close" aria-label="Close" (click)="modal.dismiss()"></button>
+    </div>
+    <div class="modal-body">
+        <table class="table table-borderless">
+            <tbody>
+                @if (rawBuildLogs.length === 0) {
+                    <tr class="build-output__entry">
+                        <td class="build-output__entry-text">
+                            <span jhiTranslate="artemisApp.buildQueue.logs.noLogs"></span>
+                        </td>
+                    </tr>
+                } @else {
+                    @for (logEntry of rawBuildLogs; track logEntry) {
+                        <tr class="build-output__entry">
+                            <td class="build-output__entry-date">{{ logEntry.time | artemisDate: 'long' : true : undefined : false : true }}</td>
+                            <td class="build-output__entry-text">
+                                @for (line of logEntry.logLines; track line) {
+                                    <span>{{ line }}</span>
+                                    <br />
+                                }
+                            </td>
+                        </tr>
+                    }
+                }
```

Comment:
There is some syntactical sugar for the empty case, does not need to be changed, might just be handy for future PRs 😄 

```suggestion
                @for (logEntry of rawBuildLogs; track logEntry) {
                    <tr class="build-output__entry">
                        <td class="build-output__entry-date">{{ logEntry.time | artemisDate: 'long' : true : undefined : false : true }}</td>
                        <td class="build-output__entry-text">
                            @for (line of logEntry.logLines; track line) {
                                <span>{{ line }}</span>
                                <br />
                            }
                        </td>
                    </tr>
                } @empty {
                    <tr class="build-output__entry">
                        <td class="build-output__entry-text">
                            <span jhiTranslate="artemisApp.buildQueue.logs.noLogs"></span>
                        </td>
                    </tr>
                }
```